src/containers/Main/VrtConversion.tsx

```diff
@@ -0,0 +1,208 @@
+        setUserEnabled(new BigNumber(userVrtAllowanceTemp).gt(0));
+        setVrtConverterXvsBalance(
+          new BigNumber(vrtConverterXvsBalanceTemp).div(XVS_DECIMAL),
+        );
+      }
+    };
+
+    update();
+
+    return () => {
+      mounted = false;
+    };
+  }, [fastRefresh, account]);
+
+  return (
+    <MainLayout title="Redeem VRT">
+      {loading ? <LoadingSpinner /> : (
+        <VrtConversionWrapper>
+          <Row className="vrt-conversion-container">
+            <Col
+              xl={{ span: 8, offset: 8 }}
+              lg={{ span: 12, offset: 6 }}
+              md={{ span: 12, offset: 6 }}
+              sm={{ span: 24 }}
+              xs={{ span: 24 }}
+            >
+              <div className="container">
+                <TabContainer
+                  styles={{
+                    padding: '10%',
+                    marginTop: '24px',
+                  }}
+                  titles={['Redeem', 'Withdraw']}
+                >
+                  <Redeem
+                    vrtConverterXvsBalance={vrtConverterXvsBalance}
+                    redeemableAmount={redeemableAmount}
+                    vrtConversionDailyLimit={vrtDailyLimit}
+                    vrtDailyUtilised={dailyUtilisation}
+                    userVrtBalance={userVrtBalance}
+                    userEnabled={userEnabled}
+                    conversionEndTime={conversionEndTime}
+                    conversionRatio={conversionRatio}
+                    handleClickRedeem={async redeemAmount => {
+                      try {
+                        if (!userEnabled && account) {
+                          // approve user's allownace to converter
+                          await vrtTokenContract.methods
+                            .approve(
+                              vrtConverterContract.options.address,
+                              new BigNumber(2)
+                                .pow(256)
+                                .minus(1)
+                                .toFixed(),
+                            )
+                            .send({
+                              from: account,
+                            });
+                        } else {
+                          await vrtConverterContract.methods
+                            .convert(redeemAmount.times(VRT_DECIMAL).toFixed())
+                            .send({
+                              from: account,
+                            });
+                        }
+                      } catch (e) {
+                        console.log('>> redeem error', e);
+                      }
+                    }}
+                    account={account || ''}
+                  />
+                  <Withdraw
+                    withdrawableAmount={new BigNumber(0)}
+                    handleClickWithdraw={() => {
+                      xvsVestingContract.methods.withdraw(account);
+                    }}
+                    account={account || ''}
```

Comment:
could we do something like `const { account = '' } = useWeb3React();` to avoid this operator?

Another option would be for useWeb3React to return the default

Reply:
yes, useWeb3React is used directly from an external library, we'd better add a wrapper to it, but this time I'd like to adopt option one and leave the second for later refactoring